tests/test_vtk.py

```diff
@@ -45,6 +46,36 @@ def test_pluto_units(pluto_vtk_file):
         assert_allclose_units(getattr(ds, f"{u}_unit"), expected)
 
 
+def test_pluto_units_override(pluto_vtk_file):
+    file = pluto_vtk_file
+    ug = {
+        "time_unit": (2.0, "s"),
+        "length_unit": (4.0, "cm"),
+        "mass_unit": (5.0, "kg"),
+        "density_unit": (0.078125, "kg/cm**3"),
+        "velocity_unit": (2.0, "cm/s"),
+        "magnetic_unit": (62.66570686577499, "gauss"),
+    }
+    unit_combs = distinct_combinations(ug, 3)
+    invalid = yt_idefix.PlutoVtkDataset.invalid_unit_combinations
+    for comb in unit_combs:
+        uo = {}
+        for unit in comb:
+            uo[unit] = ug[unit]
+        if set(uo) in invalid:
+            with pytest.raises(ValueError, match=r".* cannot derive all units\n.*"):
```

Comment:
I'd like the error message to be even clearer. Please explicitly say that the input units_override isn't valid

Reply:
Okay, I'll improve it

Reply:
Here I just wanted to match the error messeage to make sure it's the `ValueError` I expected, not caused by something else. I think the regular expression is enough, right?